docs/getting-started/auth.adoc

```diff
@@ -1,15 +1,169 @@
 = AeroGear Services Auth SDK
 
-Mobile authentication SDK based on Keycloak
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+Provides authentication features like access control and two factor authentication through Keycloak.
 
 == Adding dependency
 
 Add dependency to your application module
 ----
 dependencies {
-    implementation group: 'org.aerogear.android', name: 'auth'
+    implementation 'org.aerogear:auth:5.0.0-SNAPSHOT'
 }
 ----
 
 == Usage
 
+To use the Auth SDK you'll first need to:
+
+* Import the Core module. See link:./core.adoc[the Core documentation].
+* Specify the schema of the redirect url for your Android project in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file.
+It should match the schema specified in the `Valid Redirect URL` section for the client in Keycloak.
+It is recommended to use the package name of the Android app as the schema of the redirect url to avoid conflicts.
+
+=== Initializing the SDK
+
+`AuthService` can be retrieved using the `MobileCore#getInstance` method.
+
+[source,java]
+----
+AuthService authService = mobileCore.getInstance(AuthService.class);
+----
+
+Any subsequent `MobileCore#getInstance` method will return the same instance of `AuthService`.
+
+Before the `AuthService` can be used used `AuthService#init` must be invoked once in an app.
+
+If `AuthService#init` is not invoked then an `IllegalStateException` will be thrown when using the
+service.
+
+[source,java]
+----
+AuthServiceConfiguration authServiceConfig = new AuthServiceConfiguration.AuthConfigurationBuilder()
+    .withRedirectUri("org.aerogear.mobile.example:/callback")
+    .build();
+
+// Only invoke this once, every subsequent retrieval of the AuthService will retrieve the same, already initialized, instance.
```

Comment:
Maybe rephrase as 'You only need to invoke this once'.
'Only invoke this once' makes it sound dangerous, even though the end of the sentence seems to imply its fine to invoke it many times.